test/e2e/helpers/broker_test_helper.go

```diff
@@ -245,7 +251,7 @@ func TestBrokerWithManyTriggers(t *testing.T, brokerCreator BrokerCreator, shoul
 			eventTrackers := make(map[string]*recordevents.EventInfoStore, len(test.eventFilters))
 			for _, event := range test.eventFilters {
 				// Create event recorder pod and service
-				subscriberName := "dumper-" + event.String()
+				subscriberName := event.Name()
```

Comment:
Why did we remove the `"dumper-"` prefix (and `"trigger-"` one below)?  Those common prefixes tend to help me establish the mental models when trying to debug a new (to me) test.

Reply:
I removed it because the names need to be created in advance, and if I then prefix it with "dumper-" it may exceed the 63 limit, plus I'm getting segmentation faults because eventTrackers then don't include the right matcher and it calls a function on `nil` object. Anyway, I'll see if we can improve it a bit.

Reply:
Right now I'm facing a problem with running tests in parallel. Some of them label namespaces with `knative-eventing-injection": "enabled"` and some of them don't want this. These tests can't run in parallel. Pretty complicated...